### PR TITLE
Negative bars

### PR DESCRIPTION
**Pull request by xybrnet**

### CODE REVIEW

File: scorecard/static/js/charts.js [hunk 1]
```diff
@@ -47,26 +47,35 @@ var Chart = function() {
         .attr("width", self.width + self.margin.left + self.margin.right)
         .attr("height", self.height + self.margin.top + self.margin.bottom)
       .append("g")
-        .attr("transform", "translate(" + self.margin.left + "," + self.margin.top + ")");
+        .attr("transform", "translate(" + self.margin.left + "," + self.margin.top + ")")
```
Comment (longhotsummer) [inline scorecard/static/js/charts.js]: don't forget your semicolons :)

File: scorecard/static/js/charts.js [hunk 2]
```diff
@@ -47,26 +47,35 @@ var Chart = function() {
...
     //  Draw the bars
     self.svg.selectAll(".bar")
         .data(data)
       .enter().append("rect")
         .attr("class", "chart-bar")
         .attr("x", function(d) { return self.x(d.year); })
         .attr("width", self.x.rangeBand())
-        .attr("y", function(d) { return self.y(d.result); })
-        .attr("height", function(d) { return self.height - self.y(d.result); })
+        .attr("y", function(d) { return d.result < 0 ? self.y(0) : self.y(d.result); })
```
Comment (longhotsummer) [inline scorecard/static/js/charts.js]: `self.y(Math.max(d.result), 0)` perhaps?

File: scorecard/static/js/charts.js (hunk 1 above)
Comment (xybrnet) [inline scorecard/static/js/charts.js]: Thanks, fixed.

File: scorecard/static/js/charts.js (hunk 2 above)
Comment (xybrnet) [inline scorecard/static/js/charts.js]: Much better.

**Pull request merged**